### PR TITLE
Add a add photo Button when user is in limited photo mode

### PR DESCRIPTION
**Pull request by ruby109**
Add a add photo Button when user is in limited photo mode
当用户在限制图片的权限模式下，显示一个加号按钮，让用户调起 PHPicker 来选择新照片，仿新浪微博的设计
IMG_1303

### CODE REVIEW

File: Sources/Extensions/Int+ZLPhotoBrowser.swift [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//
+//  Int+ZLPhotoBrowser.swift
+//  ZLPhotoBrowser
+//
+//  Created by Masadora on 2020/11/3.
+//
+
```
Comment (longitachi) [inline Sources/Extensions/Int+ZLPhotoBrowser.swift:7]: Add copyright here.

Comment (ruby109) [reply Sources/Extensions/Int+ZLPhotoBrowser.swift:26]: added :-)

**Comment by longitachi** [comment]
滑动选择的两处bug，`-1` 修改为 `-self.offset`
```
var i = beginIndexPath.row
while (minIsBegin ? i <= maxIndex : i >= minIndex) {
    if i != beginIndexPath.row {
        let p = IndexPath(row: i, section: 0)
        if !self.arrSlideIndexPaths.contains(p) {
            self.arrSlideIndexPaths.append(p)
            // let index = asc ? i : i - 1 修改为 let index = asc ? i : i - self.offset
            let index = asc ? i : i - self.offset
            let m = self.arrDataSources[index]
            self.dicOriSelectStatus[p] = m.isSelected
        }
    }
    i += (minIsBegin ? 1 : -1)
}

for path in self.arrSlideIndexPaths {
    // let index = asc ? path.row : path.row - 1 修改为 let index = asc ? path.row : path.row - self.offset
    let index = asc ? path.row : path.row - self.offset
    ...
```

File: Sources/General/ZLThumbnailViewController.swift [hunk 2]
```diff
@@ -757,12 +775,21 @@ extension ZLThumbnailViewController: UICollectionViewDataSource, UICollectionVie
             return cell
         }
         
+        if #available(iOS 14, *) {
```
[the remark this answers is not in the document]
Comment (ruby109) [reply Sources/General/ZLThumbnailViewController.swift:778]: 已提交 commit

**Pull request merged**